File: VB/UserDiffsToDB.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.BaseImpl

Namespace UserDiffsToDB.Module
    Public Class Updater
        Inherits ModuleUpdater
        Public Sub New(ByVal session As Session, ByVal currentDBVersion As Version)
            MyBase.New(session, currentDBVersion)
        End Sub
        Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
            MyBase.UpdateDatabaseAfterUpdateSchema()

            Dim adminUser As SimpleUser = Session.FindObject(Of SimpleUser)(New BinaryOperator("UserName", "Sam"))
            If adminUser Is Nothing Then
                adminUser = New SimpleUser(Session)
                adminUser.UserName = "Sam"
                adminUser.FullName = "Sam"
            End If
            adminUser.IsAdministrator = True
            adminUser.SetPassword("")
            adminUser.Save()

            Dim user As SimpleUser = Session.FindObject(Of SimpleUser)(New BinaryOperator("UserName", "John"))
            If user Is Nothing Then
                user = New SimpleUser(Session)
                user.UserName = "John"
                user.FullName = "John"
            End If
            user.IsAdministrator = False
            user.SetPassword("")
            user.Save()
        End Sub
    End Class
End Namespace

File: CS/UserDiffsToDB.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.ExpressApp;
using DevExpress.Persistent.BaseImpl;

namespace UserDiffsToDB.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();

            SimpleUser adminUser = Session.FindObject<SimpleUser>(new BinaryOperator("UserName", "Sam"));
            if (adminUser == null) {
                adminUser = new SimpleUser(Session);
                adminUser.UserName = "Sam";
                adminUser.FullName = "Sam";
            }
            adminUser.IsAdministrator = true;
            adminUser.SetPassword("");
            adminUser.Save();

            SimpleUser user = Session.FindObject<SimpleUser>(new BinaryOperator("UserName", "John"));
            if (user == null) {
                user = new SimpleUser(Session);
                user.UserName = "John";
                user.FullName = "John";
            }
            user.IsAdministrator = false;
            user.SetPassword("");
            user.Save();
        }
    }
}